src/MusicTheory/Octave.elm

```diff
@@ -0,0 +1,101 @@
+module MusicTheory.Octave exposing
+    ( Octave
+    , add
+    , all
+    , down
+    , eight
+    , five
+    , four
+    , number
+    , octave
+    , one
+    , seven
+    , six
+    , three
+    , two
+    , up
+    , zero
+    )
+
+
+type Octave
+    = Octave Int
```

Comment:
I think it would be nicer to do something like
```
Octave OctaveNumber
```
where:
```
type OctaveNumber
    = One
    | Two
    | Three
    | Four
    | Five
    | Six
    | Seven
    | Eight
```
I realize we'll still be using `Maybe` or `Result` in `up` and `down`, since there are limits. But it would be nicer to make `Octave -99999` invalid.